src/utils/popoverMixins.js

```diff
@@ -58,11 +59,30 @@ export default {
         trigger.setTriggerBy(this)
       }
     },
+    clearTimeout() {
+      clearTimeout(this.delayTimeout)
+      this.delayTimeout = null
+    },
     toggle (e) {
       let trigger = getFirstChild(this.$refs.trigger)
       if (e && this.trigger === 'contextmenu' && trigger === e.target) e.preventDefault()
-      if (!(this.show = !this.show)) {
+      console.log(e.type)
+      if (this.show) {
+        if (e.type === 'mouseleave') { // only delay closing for hover events
+          this.delayTimeout = setTimeout(() => {
+            this.show = false
+            this.clearTimeout()
+          }, 200)
```

Comment:
I tested 200 was quite fast, is there a reason to choose 200? Would 300 be better?

Reply:
I made it a global variable for now, so that it can be easily tested.

```js
window.__POPOVER_TIMEOUT = xxx
```

Do the above in a console to set the timeout in milliseconds.